### PR TITLE
fix: Move extension field to the next line

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details

- Extension field label "Extension (Optional)" was overlapping with the text for smaller screen devices. 

### Fixed Issues

$ https://github.com/Expensify/App/issues/7645

### Tests
1. Tested the field in all the platforms
2. Tested the label in small screen device.

- [x] Verify that no errors appear in the JS console

### QA Steps
1. Open app
2. Navigate to Concierge chat > Click on Request a call
3. Check “extension” field. The label shouldn't overlap with the text. Ensure to check with a small screen device.

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

web-optional-label-extension

#### Mobile Web

mweb-small-optional-label-extension-field
mweb-optional-label-extension-field

#### Desktop

Screenshot 2022-02-19 at 1 36 43 PM

#### iOS

Screenshot 2022-02-19 at 1 36 01 PM

#### Android

Screenshot 2022-02-19 at 1 39 54 PM

### CODE REVIEW

**Review by mountiny — COMMENTED** [review]
LGTM, gonna wait for Rajat with approval.

File: src/pages/RequestCallPage.js [hunk 1]
```diff
@@ -317,31 +317,29 @@ class RequestCallPage extends Component {
...
-                                        autoCompleteType="off"
-                                        autoCorrect={false}
-                                        value={this.state.phoneExtension}
-                                        placeholder="100"
-                                        errorText={this.state.phoneExtensionError}
-                                        onBlur={this.validatePhoneExtensionInput}
-                                        onChangeText={phoneExtension => this.setState({phoneExtension})}
-                                    />
-                                </View>
+                            <View style={[styles.flex1]}>
```
Comment (parasharrajat) [inline src/pages/RequestCallPage.js:320]: I don't think flex1 is needed so just remove this view.

File: src/pages/RequestCallPage.js [hunk 2]
```diff
@@ -317,31 +317,29 @@ class RequestCallPage extends Component {
...
+                                    autoCompleteType="off"
+                                    autoCorrect={false}
+                                    value={this.state.phoneNumber}
+                                    placeholder="2109400803"
+                                    errorText={this.state.phoneNumberError}
+                                    onBlur={this.validatePhoneInput}
+                                    onChangeText={phoneNumber => this.setState({phoneNumber})}
+                                />
+                            </View>
+                            <View style={[styles.mt4, styles.flex1]}>
```
Comment (parasharrajat) [inline src/pages/RequestCallPage.js:332]: we can remove this view as well and add the mt4 to text input and remove flex1.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by mananjadhav** [comment]
@parasharrajat PR updated

**Review by parasharrajat — APPROVED** [review]
LGTM. 

CC: @mountiny 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by mountiny — APPROVED** [review]
Thank you both!

**Pull request merged**